src/components/Lakritzel/ProductCard.js

```diff
@@ -169,6 +169,7 @@ const Headline = styled.h2`
 `
 
 const Details = styled.ol`
+  font-family: var(--main-font);
```

Comment:
Instead of adding the font-family for all components, you could declare the font-family for `body` and only change the font-family where necessary. This could reduce some code I think. 🤔